pydantic_graph/README.md

```diff
@@ -0,0 +1,16 @@
+# Pydantic Graph
+
+[![CI](https://github.com/pydantic/pydantic-ai/actions/workflows/ci.yml/badge.svg?event=push)](https://github.com/pydantic/pydantic-ai/actions/workflows/ci.yml?query=branch%3Amain)
+[![Coverage](https://coverage-badge.samuelcolvin.workers.dev/pydantic/pydantic-ai.svg)](https://coverage-badge.samuelcolvin.workers.dev/redirect/pydantic/pydantic-ai)
+[![PyPI](https://img.shields.io/pypi/v/pydantic-graph.svg)](https://pypi.python.org/pypi/pydantic-graph)
+[![versions](https://img.shields.io/pypi/pyversions/pydantic-graph.svg)](https://github.com/pydantic/pydantic-ai)
+[![license](https://img.shields.io/github/license/pydantic/pydantic-ai.svg?v)](https://github.com/pydantic/pydantic-ai/blob/main/LICENSE)
+
+Graph and finite state machine library.
+
+This library is developed as part of the [PydanticAI](https://ai.pydantic.dev), however it has no dependency
+on `pydantic-ai` or related packages and can be considered as a pure graph library.
```

Comment:
```suggestion
on `pydantic-ai` or related packages and can be considered as a pure graph-based state machine library.
```
(I think I made a similar suggestion the first time I encountered this phrasing earlier in the PR.)